docs/api/non_geo_datasets.csv

```diff
@@ -2,6 +2,7 @@ Dataset,Task,Source,# Samples,# Classes,Size (px),Resolution (m),Bands
 `ADVANCE`_,C,"Google Earth, Freesound","5,075",13,512x512,0.5,RGB
 `Benin Cashew Plantations`_,S,Airbus Pléiades,70,6,"1,122x1,186",10,MSI
 `BigEarthNet`_,C,Sentinel-1/2,"590,326",19--43,120x120,10,"SAR, MSI"
+`Cloud Cover Detection`_,S,Sentinel-2,"22,728",2,512x512,10,MSI
```

Comment:
```suggestion
`CloudCover`_,S,Sentinel-2,"22,728",2,512x512,10,MSI
```
This name needs to match the name in `docs/api/datasets.rst`. Alternatively, you can change that name instead.